Source/Core/Credit.js

```diff
@@ -2,61 +2,42 @@ define([
     './defaultValue',
     './defined',
     './defineProperties',
-    './DeveloperError'
+    './deprecationWarning',
+    './DeveloperError',
+    '../ThirdParty/xss'
 ], function(
     defaultValue,
     defined,
     defineProperties,
-    DeveloperError) {
+    deprecationWarning,
+    DeveloperError,
+    xss) {
     'use strict';
 
     var nextCreditId = 0;
     var creditToId = {};
 
-    function getElement(credit) {
-        var element = document.createElement('span');
-        var text = credit.text;
-        var link = credit.link;
-        var a;
-        if (credit._hasImage) {
-            var content = document.createElement('img');
-            content.src = credit.imageUrl;
-            if (defined(text)) {
-                content.alt = text;
-                content.title = text;
-            }
-            if (credit._hasLink) {
-                a = document.createElement('a');
-                a.appendChild(content);
-                a.href = link;
-                a.target = '_blank';
-                element.appendChild(a);
-            } else {
-                element.appendChild(content);
-            }
-            element.className = 'cesium-credit-image';
-        } else {
-            if (credit._hasLink) {
-                a = document.createElement('a');
-                a.textContent = text;
-                a.href = link;
-                a.target = '_blank';
-                element.appendChild(a);
-            } else {
-                element.textContent = text;
-            }
-            element.className = 'cesium-credit-text';
+    function createDomNode(html) {
+        var div = document.createElement('span');
```

Comment:
I don't think this can't be a `span` because a span technically can't have child elements.

Reply:
> a span technically can't have child elements

Are you sure?  I couldn't find any sources that say that.  I use a `span` because it's inherently `inline`, where a `div` is a block element.  Plus we were putting everything into a `span` already and it's worked fine.

Reply:
Here's a good answer on the subject with links back to the spec https://stackoverflow.com/a/11314736

`span` can only have "phrasing elements", so I was only half-right but using a div is still the correct way to go.  If it's trivial to use an inline div, then we should, but we could probably get away with a span element if we had to.